ci/run-tests.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -eo pipefail
+
+ROOT="$(cd "$(dirname "$0")/.." && pwd)"
+
+main() {
+	echo "** Starting xvfb"
+	export DISPLAY=:99
+	/usr/bin/Xvfb $DISPLAY &
+	echo "** Started xvfb"
```

Comment:
I just blindly follow the [Google shell style guide](https://google.github.io/styleguide/shell.xml) for all scripts (so specifically [this](https://google.github.io/styleguide/shell.xml?showone=Indentation#Indentation)).